diff --git a/dist/content/loaders/file.js b/dist/content/loaders/file.js
index 2131e9c1790a5f3d931938fba41b681777b13f14..cce006d902df92935be9fb428ad3155892513abb 100644
--- a/dist/content/loaders/file.js
+++ b/dist/content/loaders/file.js
@@ -15,6 +15,7 @@ function file(fileName) {
       logger.debug(error.message);
       return;
     }
+    const normalizedFilePath = posixRelative(fileURLToPath(settings.config.root), filePath);
     if (Array.isArray(json)) {
       if (json.length === 0) {
         logger.warn(`No items found in ${fileName}`);
@@ -31,7 +32,7 @@ function file(fileName) {
         store.set({
           id,
           data,
-          filePath: posixRelative(fileURLToPath(settings.config.root), filePath)
+          filePath: normalizedFilePath,
         });
       }
     } else if (typeof json === "object") {
@@ -40,7 +41,7 @@ function file(fileName) {
       store.clear();
       for (const [id, rawItem] of entries) {
         const data = await parseData({ id, data: rawItem, filePath });
-        store.set({ id, data });
+        store.set({ id, data, filePath: normalizedFilePath });
       }
     } else {
       logger.error(`Invalid data in ${fileName}. Must be an array or object.`);
