diff --git a/dist/assets/services/sharp.js b/dist/assets/services/sharp.js
index 368079496916375790827ceaa064a2bc733bcc0b..73e5e2fdea5daee1c670c1a93aa40135620a5bac 100644
--- a/dist/assets/services/sharp.js
+++ b/dist/assets/services/sharp.js
@@ -91,7 +91,7 @@ const sharpService = {
     }
     const result = sharp(inputBuffer, {
       failOnError: false,
-      pages: -1,
+      // pages: -1,
       limitInputPixels: config.service.config.limitInputPixels
     });
     result.rotate();
