diff --git a/dist/assets/services/sharp.js b/dist/assets/services/sharp.js
index 9078923a98fa1f63c7550a2ad6d12ff52c6cfce5..58b44c0b6e90bcc73841d562827c44dcf3bf6d11 100644
--- a/dist/assets/services/sharp.js
+++ b/dist/assets/services/sharp.js
@@ -84,7 +84,7 @@ const sharpService = {
     if (transform.format === "svg") return { data: inputBuffer, format: "svg" };
     const result = sharp(inputBuffer, {
       failOnError: false,
-      pages: -1,
+      // pages: -1,
       limitInputPixels: config.service.config.limitInputPixels
     });
     result.rotate();
