diff --git a/dist/assets/services/sharp.js b/dist/assets/services/sharp.js
index 65f1e50c6a4dcafef4830bbfe3421bb8f783284d..f02e62cc8b19079324bb6ba8f003cfb42d8f4a51 100644
--- a/dist/assets/services/sharp.js
+++ b/dist/assets/services/sharp.js
@@ -111,7 +111,7 @@ const sharpService = {
     }
     const result = sharp(inputBuffer, {
       failOn: "none",
-      pages: -1,
+      // pages: -1,
       limitInputPixels: config.service.config.limitInputPixels
     });
     result.rotate();
